clarify #500 (#513)
solidjs/solid-docs-next: src/routes/concepts/understanding-jsx.mdx

  They let components react in real-time to changes, resulting in a responsive application.

      <Callout>

>       	Expressions, whether fixed or dynamic, get applied *in the order defined within the JSX*. This works for a wide range of DOM elements, but won't work with elements that require attributes to be defined in a special order, like input types with `type='range'`.

Reviewer: ```suggestion
      	Expressions, whether fixed or dynamic, get applied *in the order defined within the JSX*. 
      	This works for a wide range of DOM elements, but will not work with elements that require attributes to be defined in a special order, such as input types with `type='range'`.
```

>   When order influence the element's behavior, users must define the expressions in the order that the element expect.

Reviewer: ```suggestion
  When order influences an element's behavior, users must define the expressions in the order that the elements are expected.
```

Author: Thanks Sarah!
